site/src/patterns/accordion.md

```diff
@@ -0,0 +1,164 @@
+---
+title: Accordion
+description: Accordion patterns in the Atlas Design System
+template: standard
+---
+
+# Accordion patterns
+
+Because the `details` component has so many possible permutations, it can be helpful to provide examples of certain styles.
+
+## Accordion
```

Comment:
Can you provide a brief title and explanation for every pattern? Check the cards pattern page for the reference. Right now it's just a pile of code which kinda hard to go through and understand what are the scenarios

Reply:
Added titles for the accordion patterns